modules/external-api/src/main/java/org/opencastproject/external/endpoint/EventsEndpoint.java

```diff
@@ -455,6 +455,41 @@ public Response updateEventMetadata(@HeaderParam("Accept") String acceptHeader,
           indexService.updateEventAcl(eventId, eventHttpServletRequest.getAcl().get(), externalIndex);
         }
 
+        if (eventHttpServletRequest.getProcessing().isSome()) {
+          if (event.isScheduledEvent() && !event.hasRecordingStarted()) {
```

Comment:
Avoid unnecessary long conditions:

```suggestion
          if (!event.isScheduledEvent() || event.hasRecordingStarted()) {
            return RestUtil.R.badRequest("Processing can't be updated for events that are already uploaded.");
          }
```